traitsui/undo.py

```diff
@@ -197,7 +201,7 @@ def merge_undo(self, undo_item):
                                     return False
                                 self.new_value = v2
                                 return True
-                        except:
+                        except Exception:
                             pass
```

Comment:
`raise_to_debug()` like the others? Or is this one legitimate?

Reply:
This one's legitimate based on the comment earlier in the function: https://github.com/enthought/traitsui/pull/510/files#diff-fbedc1554be3859074ab2232abb6ca73R185

I'd be happier if I knew what particular exception `wxColour` raises, but mainly I just care that we're not doing a bare `except` here.

Reply:
Weird: `isinstance(wx.Colour(), collections.Sequence) == False` for me, but `len(wx.Colour()) == 3`.